lib/logger/index.ts

```diff
@@ -63,11 +63,22 @@ function createLogFileStream(logFile: string): bunyan.Stream {
   const directoryName = upath.dirname(logFile);
   fs.ensureDirSync(directoryName);
 
-  return {
+  const file: bunyan.Stream = {
     name: 'logfile',
     path: logFile,
     level: validateLogLevel(getEnv('LOG_FILE_LEVEL'), 'debug'),
   };
+
+  const logFileFormat = getEnv('LOG_FILE_FORMAT');
+
+  if (
+    is.nonEmptyStringAndNotWhitespace(logFileFormat) &&
+    logFileFormat !== 'json'
```

Comment:
no, we should not support arbitrary values as valid

Reply:
So what do I do, `logFileFormat == 'pretty'`? It's just that in the other case of `LOG_FORMAT` it doesn't really check against a fixed list of formats either, it just looks to see if it's different from `json`.

That is, in the code that already is, the check is arbitrary, just look that it is different from json, not a concrete value like `pretty` for example.

https://github.com/renovatebot/renovate/blob/470f03ccfc9efbd3492a848c91ac8c8d2b0acf8d/lib/logger/index.ts#L37

Reply:
I mean that for `LOG_FORMAT` is not documented the valid values other than `json`, it should be documented the current two possible values.

Reply:
Fallback to `pretty` if the given value isn't `json`. I'm not sure if we can already warn the user here about an invalid config option. 🤔 

@rarkins WDYT? Shcouzld we exit early when an invalid value is passed? Would probably my preference but could make trouble if `LOG_FILE_FORMAT` is used for something else.